roles/openshift_grafana/tasks/gf-permissions.yml

```diff
@@ -1,12 +1,18 @@
 ---
-- name: Create gf user on htpasswd
-  command: htpasswd -c /etc/origin/master/htpasswd gfadmin
+- name: Create grafana user on htpasswd
+  htpasswd:
+    path: /etc/origin/master/htpasswd
+    name: "{{ grafana_user }}"
+    password: "{{ grafana_password }}"
 
-- name: Make sure master config use HTPasswdPasswordIdentityProvider
-  command: "sed -ie 's|AllowAllPasswordIdentityProvider|HTPasswdPasswordIdentityProvider\n      file: /etc/origin/master/htpasswd|' /etc/origin/master/master-config.yaml"
+- name: replace IdentityProvider
+  lineinfile:
```

Comment:
If modifying yaml, please use the yedit module.  `lineinfile` is generally not used on structured data.

Reply:
I don't see the yedit module listed in the latest modules: http://docs.ansible.com/ansible/latest/list_of_all_modules.html